Automattic/jetpack
# Blocks: Update keywords for better search results

Changes and adds keywords to a few blocks that will hopefully be easier to find now.

Fixes #14628.

#### Changes proposed in this Pull Request:

* Updates `map` keyword to `maps` in Map block. This will allow for both strings to return a result in the block selector.
* Adds `social` keyword to Pinterest block.
* Adds a variety of new keywords to some core blocks, hoping for better findability.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
See Quick Wins in paYJgx-qn-p2.

#### Testing instructions:






* Run `yarn build-extensions` to compile these changes.
* Open the Editor, then open the block picker and search for "social". Facebook, Twitter, Tumblr, Pinterest, and Instagram blocks should be part of the search results.
* In your browser console, run `wp.blocks.getBlockType('core-embed/crowdsignal')` and check to see that the "survey" keyword was added.

#### Proposed changelog entry for your changes:

* Enhanced block search results through additional keywords.
